turbinia/config/turbinia_config.py

```diff
@@ -94,7 +93,7 @@
 # TODO(beamcodeup): Per https://github.com/google/turbinia/issues/172, Cloud
 # Functions are only available on us-central1. Thus, hardcoding this for now.
 # Fix this when CF starts supporting more regions.
-TURBINIA_REGION = 'us-central1'
+TURBINIA_REGION = 'None'
```

Comment:
I don't mind if we set this to None instead of us-central1, but can we update the comment above as appropriate? Thanks!

Reply:
Done.